app/models/user.rb

```diff
@@ -15,4 +15,11 @@ def self.from_omniauth(auth_hash)
     user.save!
     user
   end
+
+  def last_comment_has_votes?
+    Vote.where(user_id: id)
+        .order(created_at: :desc)
+        .limit(1).first
+        .comment.votes.count > 1
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.